tests/utils/test_geometry_comparisons.py

```diff
@@ -217,6 +219,24 @@ def test_compute_relative_rotation_angle(self):
 
         np.testing.assert_allclose(computed_deg, expected_deg, rtol=1e-3, atol=1e-3)
 
+    def test_compute_relative_rotation_angle_euler(self) -> None:
+        """"Sample 100 rotations about each individual Euler angle, and ensure Scipy and GTSAM result match."""
+        num_samples = 100
+        for axis in ["x", "y", "z"]:
+            for _ in range(num_samples):
+                theta = np.random.rand() * 360
+                R_1 = Rot3(Rotation.from_euler(axis, 0, degrees=True).as_matrix())
```

Comment:
Why are we using scipy? Doesn’t gtsam rot3 have a constructor from axis angle?

Reply:
indeed GTSAM does. Scipy is just a reliable package (and a dependency we already use) so I figured it could be useful to add more coverage

Reply:
Are you implying GTSAM isn’t reliable? ;)

Reply:
I’m not concerned about the dependency, I’m concerned about doing extra work in code. Constructing the scipy rotation is unnecessary. Whoever reads this code in future would just stare and think “why would the author do that…”